qemu/tests/cfg/hv_check_cpu_utilization.cfg

```diff
@@ -5,13 +5,17 @@
     master_images_clone = image1
     remove_image_image1 =yes
     nics = ""
-    cdroms = ""
```

Comment:
Why delete this parameter?

Reply:
Before this fix ,we don't use WIN_UTILS.iso , so use cdroms = ""  to control it.
As we need use 'SetACL.exe' in WIN_UTILS.iso  for current fix, so delete cdroms = ""   to load WIN_UTILS.iso in qemu command line.

Reply:
Ok, got it, thanks.